examples/js/archive.js

```diff
@@ -16,7 +16,7 @@ let imageCount = 0;
 let fetchedFrom;
 let fetchedImages;
 let count;
-let currPagination;
+let currPagination; // currPagination is used to initiate the Paginate Class
```

Comment:
@jywarren @PeculiarE 
I believe there could be a better name for this variable but I can't seem to come up with one, what are your thoughts?

Reply:
Perhaps, you want to consider the name ''currPaginator'. So the class will be 'Paginator' and not 'Paginate'.